devel/gk/Portfile

```diff
@@ -3,16 +3,14 @@
 PortSystem              1.0
 PortGroup               github 1.0
 
-github.setup            gitkraken gk-cli 1.1.4 v
+github.setup            gitkraken gk-cli 1.5.1 v
 name                    gk
 github.tarball_from     releases
 
 categories              devel
 license                 CCBY-3
 installs_libs           no
-maintainers             {@sergiolms} \
-                        {@alejandroSuch} \
```

Comment:
Your commit message doesn't mention changing the maintainers. Did you get approval from them to do this?

Reply:
They work in my team. I'm going to be the only responsible for upgrading the binary in MacPorts from now on